Store.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
```

Comment:
@Alirezabg  

🥳 overall well done for using :
1) semantic form elements
2) using flex-boxes 

🤔 What you need to improve:
1) well-named CSS classes
2) using HTML nesting and indentation
3) using flex-box for parents elements  and work more on how to use different classes for parent and the child elements